### DISCUSSION TITLE
Image Model Support

### DISCUSSION THREAD

**Discussion by sudslab** [chenhunghan/ialacol#8] [Ideas]
Any plans to support Image Classification, Stable Diffusion, etc models?

I know LocalAI supports stable diffusion, but isn't geared directly towards k8

**Comment by chenhunghan** [comment]
> Any plans to support Image Classification, Stable Diffusion, etc models?
> 
> 
> 
> I know LocalAI supports stable diffusion, but isn't geared directly towards k8

LocalAI has an helm chart, have you tried that yet?
